nle/tests/test_nethack.py

```diff
@@ -89,6 +89,21 @@ def test_permonst(self):
         mon = nethack.permonst(1)
         self.assertEqual(mon.mname, "killer bee")
 
+        # Test https://github.com/pybind/pybind11/issues/2394
+        for _ in range(2):
+            mon = nethack.permonst(0)
+            self.assertEqual(mon.mname, "giant ant")
+
+    def test_illegal_numbers(self):
+        with self.assertRaisesRegex(IndexError, "NUMMONS"):
+            nethack.permonst(10000)
```

Comment:
Do (or should?) we have access to `NUMMONS` at runtime here? That way we could check for `upper_bound + 1`.

EDIT: better question

Reply:
Done.